tests/test_cbook.py

```diff
@@ -10,14 +10,15 @@
 from gromacs import cbook
 import gromacs.setup
 
-from .datafiles import datafile
+from datafiles import datafile
```

Comment:
Do relative imports not work anymore because it's not a package anymore? Otherwise, adding the
```python
from __future__ import absolute_import
```
and keeping the relative import would be nicer.

I look at it and think "Eww... not a relative import".

Reply:
That was an accident, fixed it. I have kept it as a package and removed it from `setup.py`